assets/js/components/setup/compatibility-checks.js

```diff
@@ -122,27 +122,47 @@ export default class CompatibilityChecks extends Component {
 
 		if ( ! installed && installURL ) {
 			return {
-				labelHTML: __( 'Install<span class="screen-reader-text"> the helper plugin</span>', 'google-site-kit' ),
+				labelHTML: createInterpolateElement(
+					__( 'Install<span> the helper plugin</span>', 'google-site-kit' ),
+					{
+						span: <span className="screen-reader-text" />,
+					}
+				),
```

Comment:
Since we're using `aria-label` for the full context of the element's label, I don't think `createInterpolateElement` is necessary here anymore.

Instead, let's pass through separate strings for `children` and `aria-label` then these objects can be simply spread into the `Link` component since they're essentially objects of props now.

E.g.
```suggestion
				children: __( 'Install', 'google-site-kit' ),
				'aria-label': __( 'Install the helper plugin', 'google-site-kit' ),
```

Reply:
@aaemnnosttv Updated. Can you check?